tests/ui/storage_class/push_constant.rs

```diff
@@ -1,12 +1,10 @@
 // Test that using push constants work.
-// NOTE(eddyb) this won't pass Vulkan validation (see `push_constant_vulkan`),
-// but should still pass the baseline SPIR-V validation.
-
 // build-pass
-
 use spirv_std as _;
 
 #[derive(Copy, Clone)]
+// `Block` decoration is required for push constants when compiling for Vulkan.
+#[cfg_attr(target_env = "vulkan1.1", spirv(block))]
```

Comment:
Will this work for other environments? I suspect not, maybe check for `unknown` instead and omit `spirv(block)` in that case?